DevelopersGuide.adoc

```diff
@@ -5986,7 +5986,7 @@ A query has to be rooted somewhere, so we've invented a root-level keyword, `:al
 In Fulcro component queries that would look like this:
 
 ```
-[{:all-people (comp/get-query Person)}]
+[{:all-friends (comp/get-query Person)}]
```

Comment:
um...this does not match the text two lines earlier...is there some other part of the guide this is inconsistent with?

Reply:
I overlooked that, I should fix that as well. But the text _below_ only refers to friends, not people.